src/search/containers/no-results/no-results.component.spec.ts

```diff
@@ -24,4 +49,33 @@ describe('NoResultsComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should unsubscribe subscriptions onDestroy', () => {
+    component.paramMapSubscription$ = new Observable().subscribe();
+    spyOn(component.paramMapSubscription$, 'unsubscribe').and.callThrough();
+
+    component.ngOnDestroy();
+    expect(component.paramMapSubscription$.unsubscribe).toHaveBeenCalled();
+  });
+
+  it('should display no results content if no error', () => {
+    component.messageId = NoResultsMessageId.NO_RESULTS;
+    activatedRoute.paramMap.subscribe(params => {
+      if (Number(params.get('id')) === NoResultsMessageId.NO_RESULTS) {
```

Comment:
@johnbenjamin-hmcts - what happens if `params.get('id')` is `null` or `undefined`? Does `Number()` handle that gracefully?